source/_integrations/bsblan.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: BSBLan
+description: Instructions on how to integrate BSBLan device into Home Assistant.
+logo: bsblan.png
+ha_category: Climate
+ha_release: 0.107
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@liudger'
+---
+
+Integrates [BSBLan](https://github.com/fredlcore/bsb_lan) device into Home Assistant.
```

Comment:
Could you add a bit of introduction text? Telling people what BSBLan is...

Reply:
Hi freak, 

Thanks for your time. I hope I fixed the documentation now. 
I still could need some help with the integration. Can't get passed the coverage error. Any suggestion are welcome!

Reply:
That is a different PR pending for review, will be picked up.